labs/2-perpetual-exchange/2-trading.md

```diff
@@ -92,6 +92,6 @@ Jupiter Perpetuals Trading seamlessly integrates Jupiter Swap. Consequently, whe
 
 The process of opening, closing, or editing a position on Jupiter Perpetual Exchange entails two transactions:
 
-- Users initiate the first transaction to request actions such as opening, closing, depositing collateral, or withdrawing collateral.
+- Users initiate the first transaction to the keeper to request actions such as opening, closing, depositing collateral, or withdrawing collateral.
```

Comment:
user didn't initiate transaction to the keeper, it send onchain and the keeper monitor it